evm_loader/program/src/types/mod.rs

```diff
@@ -4,7 +4,13 @@ pub use transaction::LegacyTx;
 pub use transaction::StorageKey;
 pub use transaction::Transaction;
 pub use transaction::TransactionPayload;
+pub use tree_map::TreeMap;
+pub use vector::Vector;
 
 mod address;
 pub mod serde;
```

Comment:
The `pub mod serde;` can be removed completely, it is never usedd.

Reply:
done